src/reconstruct/reconstruction.cpp

```diff
@@ -80,11 +80,11 @@ Reconstruction::Reconstruction(std::weak_ptr<MeshBlock> wpmb, ParameterInput *pi
   // check for necessary number of ghost zones for PPM w/o fourth-order flux corrections
   if (xorder == 3) {
     int req_nghost = 3;
-    if (NGHOST < req_nghost) {
+    if (Globals::nghost < req_nghost) {
       std::stringstream msg;
       msg << "### FATAL ERROR in Reconstruction constructor" << std::endl
           << "xorder=" << input_recon
-          << " (PPM) reconstruction selected, but nghost=" << NGHOST << std::endl
+          << " (PPM) reconstruction selected, but nghost=" << Globals::nghost << std::endl
           << "Reconfigure with --nghost=XXX with XXX > " << req_nghost - 1 << std::endl;
```

Comment:
ditto Reconfigure with Rerun
A few other places as well that I'm not going to mark.